src/net/sourceforge/kolmafia/persistence/AdventureSpentDatabase.java

```diff
@@ -170,36 +176,30 @@ public static void deserialize() {
       AdventureSpentDatabase.checkZones();
     } catch (FileNotFoundException e) {
       AdventureSpentDatabase.resetTurns(false);
-      return;
-    } catch (ClassNotFoundException e) {
+    } catch (ClassNotFoundException | ClassCastException e) {
       // Found the file, but the contents did not contain a properly-serialized treemap.
       // Wipe the bogus file.
       file.delete();
       AdventureSpentDatabase.resetTurns();
-      return;
-    } catch (ClassCastException e) {
-      // Old version of the combat queue handling.  Sorry, have to delete your queue.
-      file.delete();
-      AdventureSpentDatabase.resetTurns();
-      return;
-    } catch (IOException e) {
+    } // Old version of the combat queue handling.  Sorry, have to delete your queue.
```

Comment:
this combination has put the comments in the wrong place.

Reply:
ok